in_toto/util.go

```diff
@@ -132,25 +132,15 @@ func InterfaceKeyStrings(m map[string]interface{}) []string {
 
 /*
 subsetCheck checks if all strings in a slice of strings
-can be found in a superset slice of strings.
+can be found in a Set.
 */
-func subsetCheck(subset []string, superset []string) bool {
-	// TODO: This function might be better as addition
-	// to our Set interface.
-	// We use a Go label here to break out to the outer loop
-OUTER:
-	for _, sub := range subset {
-		for _, super := range superset {
-			if sub == super {
-				continue OUTER
-			}
+func (s Set) IsSubSet(subset Set) bool {
```

Comment:
Minor performance nit: You could do a quick len check to return early (subset can't be greater than superset).